README.md

```diff
@@ -101,15 +115,56 @@ authorized: function (occ, externalUniqueNumber) {}
 
 Para invocar a tu backend enviando estos dos parámetros puedes hacer un redirect vía POST o usando XHR.
 
-Si prefieres hacer el redirect vía POST, puedes usar la librería `HttpUtil` que se incluye en este SDK. Por ejemplo:
+Para el caso del ejemplo usamos una funcion interna para hacer una llamada via POST la cual no es parte del SDK
```

Comment:
This is not easy to understand. What do you refer to by "ejemplo"? The example repository? Or the example below?

In any case, sounds like using this `HttpUtil` thing is making the example longer than it needs to be. I suggest you replace the example below to use Ajax instead of a form-based post. For example:

```
var xhr = new XMLHttpRequest();
xhr.open('POST', './transaction/commit');
xhr.setRequestHeader('Content-Type', 'application/json');
xhr.onload = function() {
    if (xhr.status === 200) {
        var response = JSON.parse(xhr.responseText);
    } else {
      // process error
    } 
};
xhr.send(JSON.stringify({
  occ: occ,
  externalUniqueNumber: externalUniqueNumber
}));
```

Reply:
(While it's not the shorter code ever, it's shorter than the one you have now)

Reply:
The idea is re directo the user to page with the transaction details and you can not make a http redirect using ajax.

I am think to put something like this:

sendHttpPostRedirect(param1, parama2);

"Where the function implement should be done by you"

Reply:
I have made the change.